processor/filterprocessor/README.md

```diff
@@ -1,22 +1,39 @@
 # Filter Processor
 
-Supported pipeline types: metrics
+Supported pipeline types: logs, metrics
 
-The filter processor can be configured to include or exclude metrics based on
-metric name in the case of the 'strict' or 'regexp' match types, or based on other
-metric attributes in the case of the 'expr' match type. Please refer to
-[config.go](./config.go) for the config spec.
+The filter processor can be configured to include or exclude:
+
+- logs, based on resource attributes using the `strict` or `regexp` match types
+- metrics based on metric name in the case of the `strict` or `regexp` match types,
+  or based on other metric attributes in the case of the `expr` match type.
+  Please refer to [config.go](./config.go) for the config spec.
+
+It takes a pipeline type, of which `logs` and `metrics` are supported, followed
+by an action:
 
-It takes a pipeline type, of which only `metrics` is supported, followed by an
-action:
 - `include`: Any names NOT matching filters are excluded from remainder of pipeline
 - `exclude`: Any names matching filters are excluded from remainder of pipeline
 
 For the actions the following parameters are required:
- - `match_type`: strict|regexp|expr
- - `metric_names`: (only for a `match_type` of 'strict' or 'regexp') list of strings or re2 regex patterns
- - `expressions`: (only for a `match_type` of 'expr') list of expr expressions (see "Using an 'expr' match_type" below)
- - `resource_attributes`: ResourceAttributes defines a list of possible resource attributes to match metrics against. A match occurs if any resource attribute matches at least one expression in this given list. 
+
+For logs:
+
+- `match_type`: `strict`|`regexp`
+- `resource_attributes`: ResourceAttributes defines a list of possible resource
+  attributes to match logs against.
+  A match occurs if any resource attribute matches at least one expression in this given list.
+
+For metrics:
+
+- `match_type`: `strict`|`regexp`|`expr`
+- `metric_names`: (only for a `match_type` of `strict` or `regexp`) list of strings
+  or re2 regex patterns
+- `expressions`: (only for a `match_type` of `expr`) list of expr expressions
```

Comment:
Consider linking from here to documentation about regex format.

Reply:
Do you mind pointing me exactly to the doc that you have in mind so that we don't go back and forth with updating this and review comments?

Reply:
I've added a link to line 38. PTAL.